cluster/selfplay/player_wrapper.sh

```diff
@@ -24,7 +24,7 @@ echo board_size: $BOARD_SIZE
 
 python3 rl_loop.py selfplay \
   --bucket_name=$BUCKET_NAME \
-  --resign-threshold=0.88 \
-  --num_readouts=900
+  --resign-threshold=0.99 \
```

Comment:
what are player_wrapper and cc_wrapper used for? and why does their value not agree with the TPU wrapper value (0.999)?

Reply:
player_wrapper is run by the python selfplay container and cc_wrapper is run by the cc selfplay container.

values harmonized.